Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -511,7 +544,12 @@ - (void)addBreadcrumbForSessionTask:(NSURLSessionTask *)sessionTask
     SentryBreadcrumb *breadcrumb = [[SentryBreadcrumb alloc] initWithLevel:breadcrumbLevel
                                                                   category:@"http"];
 
-    UrlSanitized *urlComponents = [[UrlSanitized alloc] initWithURL:sessionTask.currentRequest.URL];
+    NSURL *requestURL = sessionTask.currentRequest.URL;
+    if (requestURL == nil) {
+        // Use a fallback URL if the request URL is nil
+        requestURL = [NSURL URLWithString:@"unknown"];
```

Comment:
Instead of falling back to a dummy URL, we might want to drop the breadcrumb. Thoughts?